### PR TITLE
Added multimap article

### PR DESCRIPTION
**Pull request by Alzamer**
## Description

Added multimap article

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🇺🇸 Translation  
- [ ] 🗑️ Chore

### CODE REVIEW

File: content/docs/std/containers/maps/multimap.mdx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+#include <iostream>
+#include <string>
+#include <map>
+
+int main(){
+  std::multimap<std::string, int> potions{ {"health", 30}, {"mana", 25}, {"mana", 50}};
+
+  potions.insert(std::make_pair("health", 200));
+
+  for (auto& [key, value] : player_stats)
```
Comment (PoetaKodu) [inline content/docs/std/containers/maps/multimap.mdx:274]: There is a compilation error here (`player_stats` -> `potions`)

File: content/docs/std/containers/maps/multimap.mdx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    class Compare = std::less<Key>,
+    class Allocator = std::allocator<std::pair<const Key, T> >
+> class multimap;
+```
+
+</TabItem>
+</Tabs>
+
+The `std::multimap` is a container that stores key-value pairs, and it works pretty much as 
+std::map, the only diffrence is the keys can repeat themselves. If the keys of both elements  are the
```
Comment (PoetaKodu) [inline content/docs/std/containers/maps/multimap.mdx:50]: Two spaces after "elements"

**Review by PoetaKodu — CHANGES_REQUESTED** [review]

File: content/docs/std/containers/maps/multimap.mdx (hunk 1 above)
Comment (Alzamer) [reply content/docs/std/containers/maps/multimap.mdx:274]: Oh, copy-paste and I've overlooked it, sorry

**Comment by PoetaKodu** [comment]
Accepted

**Pull request merged**